file-formats/ints/type/zif_aff_ints_v1.intf.abap

```diff
@@ -0,0 +1,322 @@
+      "! This is a prerequisite check class
+      prereq_chk_class      TYPE c LENGTH 30,
+      "! <p class="shorttext">Scenario Technical Id</p>
+      "! This is the scenario technical ID
+      scenario_tech_id      TYPE c LENGTH 64,
+      "! <p class="shorttext">State of Intelligent Scenario</p>
+      "! This is the state of an intelligent scenario
+      state                 TYPE c LENGTH 10,
+      "! <p class="shorttext">ABAP Language Version</p>
+      "! This is the abap language version
+      abap_language_version TYPE c LENGTH 1,
+      "! <p class="shorttext">ISLM Framework Version</p>
+      "! This is the version of ISLM framework
+      ism_version           TYPE p LENGTH 4 DECIMALS 2,
+      "! <p class="shorttext">Turnkey Class</p>
+      "! This is the turnkey class
+      tkey_class            TYPE c LENGTH 30,
+      "! <p class="shorttext">Intelligent Scenario Usage Type</p>
+      "! This is the usage type of an intelligent scenario
+      usage_type            TYPE c LENGTH 10,
+      "! <p class="shorttext">OAuth 2.0 Client Profile</p>
+      "! This is the OAuth 2.0 client profile
+      oauth_profile         TYPE c LENGTH 30,
+    END OF ty_intelligent_scenario,
+
+    "! <p class="shorttext">Binding Mappings</p>
+    "! These are scenario binding mappings
+    BEGIN OF ty_binding_mappings,
+      "! <p class="shorttext">Signature Name</p>
+      "! This is signature name
+      signature_name TYPE c LENGTH 30,
+      "! <p class="shorttext">Table or View Field Name</p>
+      "! This is a table or view field name
+      from_field     TYPE c LENGTH 30,
+      "! <p class="shorttext">Table or View Field Name</p>
+      "! This is a table or view field name
+      to_field       TYPE c LENGTH 30,
+    END OF ty_binding_mappings,
+
+    "! <p class="shorttext">Dataset Bindings</p>
+    "! These are scenario dataset bindings
+    BEGIN OF ty_bindings,
+      "! <p class="shorttext">Name</p>
+      "! This is the name of the binding
+      name      TYPE c LENGTH 30,
+      "! <p class="shorttext">Reference UUID</p>
+      "! This is a reference UUID
+      reference TYPE c LENGTH 36,
+    END OF ty_bindings,
+
+    "! <p class="shorttext">Signatures</p>
+    "! These are the scenario signatures
+    BEGIN OF ty_signatures,
+      "! <p class="shorttext">Signature Name</p>
+      "! This is the signature name
+      name           TYPE c LENGTH 30,
+      "! <p class="shorttext">Signature Type</p>
+      "! This is the signature type
+      signature_type TYPE c LENGTH 32,
+      "! <p class="shorttext">Description</p>
+      "! This is the description
+      description    TYPE c LENGTH 100,
+    END OF ty_signatures,
+
+    "! <p class="shorttext">Signature Fields</p>
+    "! These are the fields of signature
+    BEGIN OF ty_signature_fields,
+      "! <p class="shorttext">Name</p>
+      "! This is the signature input name
+      input_name     TYPE c LENGTH 30,
```

Comment:
```suggestion
      input_name     TYPE zif_aff_types_v1=>ty_object_name_30,
```

Reply:
Is the `input_name` the same content as in `ty_signatures-name`?

Reply:
Yes it is same. So, merged them as mentioned previously.